### DISCUSSION TITLE
Using Create / Destroy in Mermaid Documentation

### DISCUSSION THREAD

**Discussion by JalicX** [community/community#75514] [Repositories]
### Select Topic Area

Bug

### Body

Hey there beautiful people of GitHub!

I recently experimented with some documentation and really enjoyed using Mermaid inside of Markdown.
I am not quite sure if I found a Bug at the GH MD Compiler with using `create` and `destroy` actors inside of a Mermaid sequence Diagram since these statements seem to brick the code. My example comes from the official Mermaid Docs https://mermaid.js.org/syntax/sequenceDiagram.html#actor-creation-and-destruction-v10-3-0

#### Bricked example

```mermaid
sequenceDiagram
    Alice->>Bob: Hello Bob, how are you ?
    Bob->>Alice: Fine, thank you. And you?
    create participant Carl
    Alice->>Carl: Hi Carl!
    create actor D as Donald
    Carl->>D: Hi!
    destroy Carl
    Alice-xCarl: We are too many
    destroy Bob
    Bob->>Alice: I agree
```

#### Same example but without create and destroy

```mermaid
sequenceDiagram
    Alice->>Bob: Hello Bob, how are you ?
    Bob->>Alice: Fine, thank you. And you?
    %% create participant Carl
    Alice->>Carl: Hi Carl!
    %% create actor D as Donald
    Carl->>D: Hi!
    %% destroy Carl
    Alice-xCarl: We are too many
    %% destroy Bob
    Bob->>Alice: I agree
```

I could not find any documentation, which mermaid version currently is running at Public GH or GH Enterprise

Does somebody know so?

Have a nice day :D

**Comment by michalc** [comment]
> I could not find any documentation, which mermaid version currently is running at Public GH or GH Enterprise
> Does somebody know so?

Nosing around the iframe of a working Mermaid diagram, it seems to load some Javascript from https://viewscreen.githubusercontent.com/static/assets/mermaidMarkdown-524b2bd61dab9d5ac768.js, which has in its obfuscated source:

```javascript
const W = "10.0.2"
```
This seems to match up with a Mermaid version: https://github.com/mermaid-js/mermaid/releases/tag/v10.0.2 from March this year (2023).

So it doesn't really look this is due to a bug, but more due to GitHub running a version of Mermaid that doesn't support participant creation or destruction. Looking at https://mermaid.js.org/syntax/sequenceDiagram.html#actor-creation-and-destruction-v10-3-0 this would need version 10.3.0 or later of Mermaid.

(I too would love for this feature to work btw)
